examples/nextjs/.babelrc

```diff
@@ -1,4 +1,4 @@
 {
   "presets": ["next/babel", "@emotion/babel-preset-css-prop"],
-  "plugins": []
+  "plugins": ["@emotion"]
```

Comment:
Adding the plugin is unnecessary since the preset already includes the plugin: https://emotion.sh/docs/@emotion/babel-preset-css-prop

Reply:
Nice! Removed.